services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/QueryEnhancedRequest.java

```diff
@@ -274,39 +277,28 @@ public Builder filterExpression(Expression filterExpression) {
         }
 
         /**
-         * <p>
-         * A string that identifies one or more attributes to retrieve from the table. These attributes can include
-         * scalars, sets, or elements of a JSON document. The attributes in the expression must be separated by commas.
-         * </p>
-         * <p>
-         * If no attribute names are specified, then all attributes will be returned. If any of the requested attributes
-         * are not found, they will not appear in the result.
-         * </p>
-         * <p>
-         * For more information, see <a href=
-         * "https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.AccessingItemAttributes.html"
-         * >Accessing Item Attributes</a> in the <i>Amazon DynamoDB Developer Guide</i>.
-         * </p>
+         * The list of the attributes's paths to be retrieved from the database.
          *
-         * @param projectionExpression
-         *        A string that identifies one or more attributes to retrieve from the table. These attributes can
-         *        include scalars, sets, or elements of a JSON document. The attributes in the expression must be
-         *        separated by commas.</p>
-         *        <p>
-         *        If no attribute names are specified, then all attributes will be returned. If any of the requested
-         *        attributes are not found, they will not appear in the result.
-         *        </p>
-         *        <p>
-         *        For more information, see <a href=
-         *        "https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.AccessingItemAttributes.html"
-         *        >Accessing Item Attributes</a> in the <i>Amazon DynamoDB Developer Guide</i>.
+         * @param attributesToProject
+         *        The list of the attributes's paths to be retrieved from the database.
          * @return Returns a reference to this object so that method calls can be chained together.
          */
-        public Builder projectionExpression(String projectionExpression) {
-            this.projectionExpression = projectionExpression;
+        public Builder attributesToProject(List<String> attributesToProject) {
+            this.attributesToProject.addAll(attributesToProject);
             return this;
         }
 
+        /**
+         * The list of the attributes's paths to be retrieved from the database.
```

Comment:
'path' is a new term we have not used anywhere else in the library and may cause confusion. Can we simply refer to this as 'attribute names'? I think a little more description or at least a link to the official DynamoDB documentation would also help people understand what to expect from this feature. (Please apply this comment to other places it's applicable too)